include/swift/Sema/ConstraintSystem.h

```diff
@@ -988,8 +988,12 @@ enum ScoreKind: unsigned int {
   /// ambiguity tie-breakers should go after this; anything else
   /// should be added above.
   SK_UnappliedFunction,
+  /// A type with a missing conformance(s) that has be synthesized
+  /// or diagnosed later, such types are allowed to appear in
+  /// a valid solution.
+  SK_MissingSynthesizableConformance,
```

Comment:
Should this go before `SK_UnappliedFunction`?

Reply:
I think this is lower impact than unapplied function (property with a missing conformance is still better than unapplied function?) and for ranking we’ve got a flag and a check in compare solutions.

Reply:
We discussed this offline and came to the conclusion that it's better to move it up and filter first on Sendability.